data/json/items/grenades.json

```diff
@@ -23,6 +23,7 @@
     "type": "GENERIC",
     "name": "tear gas grenade",
     "price": 3600,
+	"flags": [ "GRENADE" ],
```

Comment:
This line isn't indented correctly, but the "GRENADE" flag should be added to the flags list in grenade_canister and shouldn't be here.